projects/power_distribution/inc/pin_defs.h

```diff
@@ -80,6 +80,12 @@
 #define FRONT_PIN_5V_SPARE_DSEL \
   { .i2c_address = POWER_DISTRIBUTION_I2C_ADDRESS_0, .pin = PCA9539R_PIN_IO0_3 }
 
+// fan ctrl pins - must be updated by HW
+#define FRONT_PIN_FAN_POT \
```

Comment:
Is this the best place for fan ctrl pin defs or should i move them to the fan_ctrl_defs.h file?